svc/pkg/game-user/ops/recommend/tests/integration.rs

```diff
@@ -1,6 +1,93 @@
 use chirp_worker::prelude::*;
+use proto::backend::{self, pkg::*};
+
+use game_user::create::Response as GameUser;
+
+async fn create_game_users(
+	ctx: &TestCtx,
+	count: usize,
+	namespace_id: Option<Uuid>,
+) -> (Uuid, Vec<GameUser>) {
+	let mut users = vec![];
+
+	let namespace_id = namespace_id.unwrap_or_else(Uuid::new_v4);
+
+	println!("Creating {count} game users...");
```

Comment:
We should be using [tracing](https://tracing.rs/tracing/) for all logging, including in tests.

Reply:
Done!